docs/Projects/TAD/ADRs/0004-software-stack.md

```diff
@@ -45,15 +46,17 @@ The Python backend will use the following key dependencies:
 
 * [Granian](https://github.com/emmett-framework/granian) as HTTP server
 * [Pydantic](https://docs.pydantic.dev/latest/) for data validation
-* TODO: decide on [FastAPI](https://fastapi.tiangolo.com/) or
-  [Flask](https://flask.palletsprojects.com/en/3.0.x/)
+* [FastAPI](https://fastapi.tiangolo.com/) for REST/API/HTML communication
+* [JinjaX](https://jinjax.scaletti.dev/) for templating
```

Comment:
I wasn't aware of JinjaX, looks nice!

Reply:
Agreed, however in the group discussion there was more support for Jinja2 is it is more widely adopted.